openshift/openshift_transport.go

```diff
@@ -57,7 +57,7 @@ func ParseReference(ref string) (types.ImageReference, error) {
 	}
 	tagged, ok := r.(reference.NamedTagged)
 	if !ok {
-		return nil, errors.Errorf("invalid image reference %s, %#v", ref, r)
+		return nil, errors.Wrapf(ok, "invalid image reference %s, %#v", ref, r)
```

Comment:
`ok` is not an `error` value (as shown by the failing tests); this should probably say either `missing tag` or `expected format: hostname/namespace/stream:tag` as in the other error message.